tests/__init__.py

```diff
@@ -0,0 +1,3 @@
+import os
+
+os.environ["AWS_DEFAULT_REGION"] = "us-east-1"
```

Comment:
oof, can't set this in travis?

Reply:
and then have local tests not pass? don't see the point